src/projects/create-pull-request/create-pull-request.html

```diff
@@ -202,6 +204,7 @@
           this.$.errorToast.show({text: 'Missing pull request description'});
           return;
         }
+        this.loadingPR = true;
```

Comment:
Instead of manually setting, you can bind to the loading property of `createPull`, see https://elements.polymer-project.org/elements/iron-ajax#property-loading